Makefile

```diff
@@ -112,16 +124,24 @@ manifest-templator: $(GO)
 
 docker-build: docker-build-operator docker-build-registry
 
-docker-build-operator: manager manifest-templator
-	$(OCI_BIN) build -f build/operator/Dockerfile -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) .
+docker-build-operator:
+ifeq ($(OCI_BIN),podman)
+	$(MAKE) build-multiarch-operator-podman
+else ifeq ($(OCI_BIN),docker)
+	$(MAKE) build-multiarch-operator-docker
+else
+	$(error Unsupported OCI_BIN value: $(OCI_BIN))
+endif
 
 docker-build-registry:
 	$(OCI_BIN) build -f build/registry/Dockerfile -t $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG) .
 
 docker-push: docker-push-operator docker-push-registry
 
 docker-push-operator:
-	$(OCI_BIN) push ${TLS_SETTING} $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
+ifeq ($(OCI_BIN),podman)
```

Comment:
This is called docker-push-operator but uses podman?

Reply:
@ashokpariya0 Can you comment on this?

Reply:
Yes, We may need to modify the section at Makefile lines 113-126(https://github.com/kubevirt/cluster-network-addons-operator/blob/main/Makefile#L113-L126), but this can be addressed in a separate PR, as renaming might impact some build processes or jobs. The existing code already supports both Docker and Podman, as indicated in [Makefile line 14](https://github.com/kubevirt/cluster-network-addons-operator/blob/main/Makefile#L14).

Reply:
With your change this target will only push in case `OCI_BIN` is equal to `podman`, IIUC?

Reply:
Yes exactly, with docker buildx, push is combined with image build.

Reply:
Fine for me, @oshoval wdyt?

Reply:
fine by me to address naming on follow-up
thx

Reply:
@oshoval  I've already looked into this and found that `docker-*` is being called from various scripts in the automation, hack folder, etc. Additionally, we need to update the README and other related files. Including all of this in a single commit would add unnecessary complexity. It would be better to handle these changes separately.

Reply:
ok thanks

Reply:
@oshoval I will create an separate issue for this, resolve it, and provide a follow-up.